src/components/Modals/CartModal.js

```diff
@@ -0,0 +1,33 @@
+import {
+  BackGround,
+  ModalContainer,
+  ConfirmRow,
+  ClearAll,
+  RemainAll,
+  ModalTitle,
+  ModalBody,
+} from "./styled";
+
+const CartModal = ({ showModal, deleteAll, noChange }) => {
```

Comment:
I am still not happy with your naming
const Modal = ({show, onOk, onCancel,title, body, okText, cancelText})